packages/code-du-travail-api/src/server/routes/search/facets.elastic.js

```diff
@@ -4,8 +4,7 @@ function getFacetsBody({ query }) {
     aggregations: {
       document_count: {
         terms: {
-          field: "source",
```

Comment:
aggregations / facet is no longer used, no ?

Reply:
It not necessary anymore indeed. But I didn't talk to anyone about this one so I kept it but if you're ok with it, I will remove it !